client/flutter/lib/api/analytics_util.dart

```diff
@@ -0,0 +1,8 @@
+import 'package:html/parser.dart';
+
+class AnalyticsUtil {
+  static String cleanValue(String text) {
```

Comment:
Please use a more descriptive name
```suggestion
  static String cleanEventParameterValue(String text) {
```

Reply:
> Please use a more descriptive name

Following your comment on https://github.com/WorldHealthOrganization/app/pull/968#discussion_r419143879, this class is gone and was replaced with logging indexes to avoid issues due to localization: 0d87f3bcc74ff3f9745f26853670cb06cc8cd1f5